forms-flow-web/src/components/Form/List.scss

```diff
@@ -116,24 +107,17 @@ flex-wrap: wrap;
     margin: 10px;
     height: 30px;white-space: nowrap;
 }
-.progress-bar {
-    background-color: #ff0000ab;
-}
-.fileupload-fail {
-    color: #f85757;
-    float: right;
-    font-weight: normal
-}
 
 .progress-bar {
-    background-color: #ff0000ab;
+    background-color: var(--color-red);
 }
 
 .fileupload-fail {
-    color: #f85757;
+    color: var(--color-failure);
```

Comment:
don't need to bring color for failure because we can use red color or color-error or color-danger

Reply:
updated in the new commit. failure is renamed with error.